stable/sysdig/values.yaml

```diff
@@ -1,5 +1,9 @@
 # Default values for Sysdig Monitor Helm package.
 
+kubernetes:
```

Comment:
Nitpick - How about using the following variable names? https://github.com/kubernetes/charts/blob/master/stable/jenkins/values.yaml#L120